UI/ui_tips.csv

```diff
@@ -392,22 +392,22 @@ Dead_DragonSummoner_002#0,"""When the Goronzoran raises its stick, it's about to
-Dead_TransformMechaA_001#0,"""トランマイザーは両腕で叩きつける攻撃の後、<br>弱点のコアが露出する。背後を狙って攻撃しよう。"""
-Dead_TransformMechaA_002#0,"""トランマイザーのミサイル攻撃はこちらを追いかけてくる。<br>攻撃を行わず、移動し続けて回避しよう。"""
-Dead_TransformMechaA_003#0,"""トランマイザーはミサイルによる攻撃の後、<br>弱点のコアが露出する。ミサイルを回避したら攻撃のチャンスだ。"""
+Dead_DragonGladiator_001#0,"""When Dragon Ex lifts both of their arms at the same time, it is preparing a<br>powerful attack. Run to the side of the Dragon Ex to avoid the attack."""
+Dead_DragonGladiator_002#0,"""When Dragon Ex stabs their sword into the ground, a shard of light pierces from the ground. You can keep running to avoid this."""
+Dead_DragonGladiator_003#0,"""Dragon Ex's shields on its arms can be destroyed.<br>If they are destroyed, the Dragon Ex's swords won't reach as far."""
+Dead_DragonGladiator_004#0,"""Poison is extremely effective against Dragon Ex's head, shields, and tail.<br>When using poison, it's easy to do lots of damage to break these parts."""
+Dead_TransformMechaA_001#0,"""After Transmizer slams the ground with both arms, its core is revealed.<br>You can attack this weak point to deal additional damage."""
+Dead_TransformMechaA_002#0,"""Transmizer's missiles try to home in on you. If you keep moving,<br>, you can avoid the missiles."""
+Dead_TransformMechaA_003#0,"""After Transmizer fires its missiles, its core is revealed.<br>You can attack this weak point to deal additional damage."""
 Dead_TransformMechaA_004#0,"""トランマイザーのマシンガン攻撃は、ジャンプしながら<br>左右に回りこんで回避しよう。"""
-Dead_BossMechaA_001#0,"""ビッグヴァーダーの甲板に上がる前に、側面の砲台を全て破壊しよう。"""
-Dead_BossMechaA_002#0,"""ビッグヴァーダーの後方にある火炎放射器は正面に立つと危険なので<br>側面から破壊するようにしよう。"""
-Dead_BossMechaA_003#0,"""ビッグヴァーダーの甲板に登った後は、後方のミサイルポッドと<br>本体肩のミサイルポッドを破壊しよう。"""
+Dead_BossMechaA_001#0,"""You should destroy the Big Vardha's guns on the sides before climbing to the deck."""
+Dead_BossMechaA_002#0,"""The flamethrowers behind Big Vardha are dangerous when attacking from the front.<br>Try attacking them from the side."""
+Dead_BossMechaA_003#0,"""The Big Vardha has missile arrays that are used to shoot you when<br>you are on the deck. You should destory them to stop the missiles."""
 Dead_BossMechaA_004#0,"""ビッグヴァーダー本体が放つ赤い電撃はジャンプでかわそう。"""
-Dead_MachineLittleBoss_001#0,"""ヴァーダーソーマはメインブースターと3つのサブブースターを使って浮遊する。<br>サブブースターを全て破壊して、空中からの攻撃を阻止しよう。"""
-Dead_MachineLittleBoss_002#0,"""ヴァーダーソーマは3つのサブブースター、両肩のミサイルポッド<br>正面の副砲、そして主砲を破壊することができる。"""
-Dead_MachineLittleBoss_003#0,"""ヴァーダーソーマのメインブースターが青白く光ったら電撃攻撃の合図。<br>急いで離れて攻撃を回避しよう。"""
-Dead_MachineLittleBoss_004#0,"""ヴァーダーソーマは主砲からビームを放ちながら1回転する。<br>攻撃範囲は広いが、足元は安全なので近づいて回避しよう。"""
+Dead_MachineLittleBoss_001#0,"""Vardha Soma has a main booster and three sub-boosters. If you destroy the sub-boosters,<br>you can stop the Vardha Soma attacking from the air."""
+Dead_MachineLittleBoss_002#0,"""A lot of Varha Soma's parts can be destroyed. Aim to damage the three sub-boosters,<br<the two shoulder missile arrays, as well as the secondary and main guns."""
```

Comment:
This entire sentence focuses on what you can destroy. It does not tell you about the boosters, but only the front cannon and the mini turrets.